### ISSUE TITLE
nl2br() should be ignored

### ISSUE THREAD

**Issue by johnbillion** [WordPress-Coding-Standards/WordPress-Coding-Standards#448]
The following code erroneously triggers an `Expected next thing to be an escaping function` warning:

```php
echo nl2br( esc_html( $value ) );
```
